index.html

```diff
@@ -0,0 +1,144 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Currency-Converter</title>
+  </head>
+  <body>
+    <div class="texttitle">
+      <h2>Currency Converter</h2>
+    </div>
+
+    <!-- currency-rate-creator part --------------->
+
+    <div class="currency-rate-creator">
+      <form class="currency-rate-creator_form" id="brand-currency-form">
+        <fieldset>
+          <legend>Brand new currency rate ceator</legend>
+
+          <label for="base-currency">The base currency</label>
+          <input
+            type="text"
+            id="base-currency"
+            value="EUR"
+            placeholder="EUR"
+            readonly
+          />
+          <br />
+
+          <div></div>
+
+          <p class="explanations-in-forms">
+            Insert a the quote valuta and rate to be added<br
+              class="zero-rate"
+            />
+          </p>
+
+          <span>
+            <label for="quote-name">Quote name</label>
+            <input
+              type="text"
+              id="quote-name"
+              step="0.001"
+              required
+              placeholder="quote-name"
+            />
+            <div></div>
+
+            <label for="quote-rate">Quote rate</label>
+            <input
+              type="number"
+              id="quote-rate"
+              step="0.001"
+              required
+              placeholder="quote-rate"
+            />
+            <div></div>
+          </span>
+          <br />
+
+          <div class="convert-currency_buttonc-section">
+            <button class="convert-currency_button" type="submit">
+              Add currency & rate
+            </button>
+          </div>
+        </fieldset>
+      </form>
+    </div>
+<BR></BR>
+
+    <!--valuta-converter part --------->
+    <div class="valuta-converter-based-on-rates">
+      <form
+        class="valuta-converter-based-on-rates_form"
+        id="valuta-converter-form"
+      >
+        <fieldset>
+          <legend>Money Conversion based on predefined exchange rate</legend>
+          <p>
+            <label>Money amount</label>
+            <input
+              id="money-amount"
+              type="number"
+              class="amount-field"
+              pattern=""
+              min="0"
+              step="0.001"
+              required
+              placeholder="Enter the amount 0.00"
+            />
+          </p>
+
+          <p></p>
+
+          <div>
+            <p class="explanations-in-forms">
+              Insert the quote valuta for conversion
+            </p>
+          </div>
+
+          <!--Base valuta-->
+          <label for="valuta-exchange-selction-base-valuta"
+            >The base valuta</label
+          >
+          <input
+            type="text"
+            id="base-currency"
+            value="EUR"
+            placeholder="EUR"
+            readonly
+          />
+
+          <div></div>
```

Comment:
can be `<div />`

Reply:
I tried that, but it didn't accept that.

Reply:
how so? I tried and it worked for me, `<div></div> `and `<div />` is exactly the same thing :)

Reply:
```
 </div>
+
+          <!--Base valuta-->
```

when its a self closing tag as this. The `/` should be at the end as in my example. 👆

Reply:
@katayouny